### PR TITLE
Enh/databases fixes

### PR DESCRIPTION
**Pull request by fontanierh**

### CODE REVIEW

File: front/lib/core_api.ts [hunk 1]
```diff
@@ -134,10 +134,8 @@ export type CoreAPIDatabaseTable = {
 };
 
 export type CoreAPIDatabaseRow = {
-  created: number;
-  table_id: string;
```
Comment (philipperolet) [inline front/lib/core_api.ts:138]: why were those removed?

File: front/pages/api/v1/w/[wId]/data_sources/[name]/databases/[dId]/tables/[tId]/rows/index.ts [hunk 2]
```diff
@@ -146,8 +151,8 @@ async function handler(
         });
       }
 
-      const { rows, total } = listRes.value;
-      return res.status(200).json({ rows, offset, limit, total });
+      const { rows: rowsList, total } = listRes.value;
+      return res.status(200).json({ rows: rowsList, offset, limit, total });
```
Comment (philipperolet) [inline front/pages/api/v1/w/[wId]/data_sources/[name]/databases/[dId]/tables/[tId]/rows/index.ts:155]: why the change?

**Review by philipperolet — COMMENTED** [review]
LGTM :+1:  just 2 extra confirmations 
- since the payload sent to core is not of the same type, I understand the current  prod core accepts CoreDatabaseRows[] already; so current front cannot send well-formed requests to core and this is what this PR fixes => correct?
- this was tested locally? (api call to send 2 rows, ack from core, ok in db?)

If yes/yes feel free to merge, my comments are nits/curiosity

:fire: Hyped that we draw closer to structured datasources being live :fire:

**Review by spolu — APPROVED** [review]
LGTM

Exactly what I started writing 👍 stopped at the rows naming clash that you named rows list ;-)

**Comment by spolu** [comment]
Thanks!! And sorry for missing that 🙏

File: front/pages/api/v1/w/[wId]/data_sources/[name]/databases/[dId]/tables/[tId]/rows/index.ts (hunk 2 above)
Comment (fontanierh) [reply front/pages/api/v1/w/[wId]/data_sources/[name]/databases/[dId]/tables/[tId]/rows/index.ts:155]: name clash with the `rows` variable in both the `GET` and `POST`

File: front/lib/core_api.ts (hunk 1 above)
Comment (fontanierh) [reply front/lib/core_api.ts:138]: They were removed from the type in `core` because we want to reduce the # of types we have (want to use the same type for input and output when possible).
`created` wasn't really useful so it was easier to remove it both from in & out.
`table_id` as well
